docs/testing-debugging/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Testing and debugging"
+sidebar_position: 57
```

Comment:
I feel like ~65 is a more natural position?

`Defining > Collecting > Testing > Enriching` vs `Testing > Defining > Collecting > Enriching`?

Reply:
I actually did consider this, can’t remember why I went with 57. Changed, thanks for the reminder!